cg/meta/workflow/microsalt.py

```diff
@@ -78,6 +82,33 @@ def get_case_fastq_path(self, case_id: str) -> Path:
     def get_config_path(self, filename: str) -> Path:
         return Path(self.queries_path, filename).with_suffix(".json")
 
+    def get_trailblazer_config_path(self, case_id: str) -> Path:
+        case_obj: models.Family = self.db.family(case_id)
+        order_id = case_obj.name
+        return Path(
+            self.root_dir, "results", "reports", "trailblazer", f"{order_id}_slurm_ids.yaml"
+        )
+
+    def get_application_type(self, case_id: str) -> str:
+        case_obj: models.Family = self.db.family(case_id)
+        return case_obj.links[0].sample.application_version.application.prep_category
+
+    def get_deliverables_file_path(self, case_id: str) -> Path:
```

Comment:
```suggestion
    def get_deliverables_file_path(self, case_id: str) -> Optional[Path]:
```
Looks like it can return None?